static/app/components/events/interfaces/crashContent/exception/content.tsx

```diff
@@ -141,6 +148,12 @@ export function Content({
       ? `exception-${exc.mechanism?.exception_id}`
       : undefined;
 
+    const frameSourceMapDebuggerData = sourceMapDebuggerData?.exceptions[
+      excIdx
+    ].frames.map(debuggerFrame =>
+      mapSourceMapDebuggerFrameInformation(sourceMapDebuggerData, debuggerFrame)
+    );
```

Comment:
Use `useMemo` so the array is not re-created every time.

Reply:
Do you see an easy way of doing this? I am asking because we're already one `.map()` deep which means I can't just simply useMemo here without majorly refactoring this component :/ (I believe)

Since this component basically re-renders very rarely anyhow I would just keep it like this for now. Thoughts?